packages/lib/src/application/Package.ts

```diff
@@ -24,14 +26,18 @@ export default class Package {
   }
 
   public static async deploy(txParams: TxParams = {}): Promise<Package> {
-    log.info('Deploying new Package...');
```

Comment:
Should this be a `Loggy.add("deploying")`?

Reply:
Same as with App, I added these messages in `AppProject.js`. The rationale behind this is that I didn't want to log these messages while pushing an external EVM-package, but just when you actually publish your project.